config/initializers/00_devise.rb

```diff
@@ -1,3 +1,4 @@
+include EnvironmentHandler
```

Comment:
this is how it gets included everywhere--I'm not sure if this is the best solution